data/profiles/linux.ipxe

```diff
@@ -1,4 +1,4 @@
 kernel <%=kernelUri%>
 initrd <%=initrdUri%>
-imgargs <%=kernelFile%> initrd=<%=initrdFile%> auto=true SYSLOGSERVER=<%=server%> API_CB=<%=server%>:<%=port%> BASEFS=<%=basefs%> OVERLAYFS=<%=overlayfs%> BOOTIF=01-<%=macaddress%> console=tty0 console=<%=comport%>,115200n8 <%=kargs%>
+imgargs <%=kernelFile%> initrd=<%=initrdFile%> console=tty0 rancher.password=root rancher.cloud_init.datasources=['url:http://<%=server%>:<%=port%>/api/common/templates/cloud-config.yaml']
```

Comment:
I'd prefer to actually make a new ipxe profile, maybe rancher.ipxe. We have a bunch of people using linux.ipxe for other custom tasks that we don't want to break functionality for.